src/test/__tests__/ReactTestUtils-test.js

```diff
@@ -19,286 +22,290 @@ var ReactTestUtils;
 
 describe('ReactTestUtils', () => {
   beforeEach(() => {
+    createRenderer = require('ReactShallowRenderer').createRenderer;
     PropTypes = require('prop-types');
     React = require('react');
     ReactDOM = require('react-dom');
     ReactDOMServer = require('react-dom/server');
     ReactTestUtils = require('ReactTestUtils');
   });
 
-  it('should have shallow rendering', () => {
-    class SomeComponent extends React.Component {
-      render() {
+  // Shallow renderer only implemented for Fiber in 16+
+  if (ReactDOMFeatureFlags.useFiber) {
```

Comment:
Yeah, totally understood- and I'm glad you bought it up. I'll figure out a strategy for www tests with stack once the fiber bundles and renderer is solid. 😁